### PR TITLE
Better NoZoom and Recoil crosshairs

### PR DESCRIPTION
**Pull request by blackMasterLord**

### CODE REVIEW

File: Osiris/Hacks/Misc.cpp [hunk 1]
```diff
@@ -317,42 +317,21 @@ void Misc::spectatorList() noexcept
...
-
-    // top (left with swapped x/y offsets)
-    drawList->AddRectFilled(ImVec2{ pos.x - 1, pos.y - 11 }, ImVec2{ pos.x + 2, pos.y - 3 }, color & IM_COL32_A_MASK);
-    drawList->AddRectFilled(ImVec2{ pos.x, pos.y - 10 }, ImVec2{ pos.x + 1, pos.y - 4 }, color);
-
-    // bottom (right with swapped x/y offsets)
-    drawList->AddRectFilled(ImVec2{ pos.x - 1, pos.y + 4 }, ImVec2{ pos.x + 2, pos.y + 12 }, color & IM_COL32_A_MASK);
-    drawList->AddRectFilled(ImVec2{ pos.x, pos.y + 5 }, ImVec2{ pos.x + 1, pos.y + 11 }, color);
+    drawList->AddCircleFilled(pos, miscConfig.recoilCrosshair.thickness, color, 360);
+    drawList->AddCircle(pos, miscConfig.recoilCrosshair.thickness, IM_COL32_BLACK, 360, 0.5);
```
Comment (R3nzTheCodeGOD) [inline Osiris/Hacks/Misc.cpp:317]: **The default crosshair is almost invisible, add +1 tickness to it by default**
`Example:`

```cpp
drawList->AddCircleFilled(pos, miscConfig.recoilCrosshair.thickness + 1, color, 360);
drawList->AddCircle(pos, miscConfig.recoilCrosshair.thickness + 1, IM_COL32_BLACK, 360, 0.5);
```

**Review by R3nzTheCodeGOD — CHANGES_REQUESTED** [review]
**Thanks for your contribution**

File: Osiris/Hacks/Misc.cpp (hunk 1 above)
Comment (blackMasterLord) [reply Osiris/Hacks/Misc.cpp:317]: This is not necessary as it can be configured in the game

**Comment by wemanzoz** [comment]
maybe an option to change recoil crosshair sizes, thickness and color? if possible

**Comment by R3nzTheCodeGOD** [comment]
@wemanzoz 
This feature already exists, but when the thickness value is 1.0, you will see the black ring, not the color, because it is too small, the default thickness value should be increased. or set the tickness value to at least 2.0

Each time you increase the thickness value, you will see that the color you selected is inside the black ring.

**Comment by wemanzoz** [comment]
> @wemanzoz 
> This feature already exists, but when the thickness value is 1.0, you will see the black ring, not the color, because it is too small, the default thickness value should be increased. or set the tickness value to at least 2.0
> 
> Each time you increase the thickness value, you will see that the color you selected is inside the black ring.

how to change the thickness value if i may ask?

**Comment by wemanzoz** [comment]
@blackMasterLord is it possible for you to make it like "toggle anti obs noscope crosshair, or engine crosshair (not anti obs)

**Pull request closed without merging**